filter.imp

```diff
@@ -1,13 +1,119 @@
 [
+   # If an architecture does not provide its own copy of asm/foo.h, then it will get an
+   # asm-generic/foo.h which can still be included as asm/foo,h
+   # This list was generated from include/uapi/asm-generic/Kbuild
+   { include: ["@.asm-generic/auxvec.h.", private, "<asm/auxvec.h>", "public"] },
+   { include: ["@.asm-generic/bitsperlong.h.", private, "<asm/bitsperlong.h>", "private"] },
+   { include: ["<asm/bitsperlong.h>", private, "<linux/bits.h>", "public"] },
```

Comment:
you should move asm/*.h -> linux/*.h rules to the third section at the bottom.

This first table should only contain asm-generic/*.h -> asm/*.h mappings.